src/pages/Dashboard/Modals/Announcement/index.tsx

```diff
@@ -0,0 +1,55 @@
+/** @jsxImportSource @emotion/react */
+import { NoticeWarning } from 'components';
+import React from 'react';
+import { useTranslation } from 'translation';
+import { Token } from 'types';
+
+import { TOKENS } from 'constants/tokens';
+
+import { useStyles } from './styles';
+
+export interface AnnouncementProps {
+  token: Token;
+}
+
+const Announcement: React.FC<AnnouncementProps> = ({ token }) => {
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+
+  // SXP disabling
+  if (token.address.toLowerCase() === TOKENS.sxp.address.toLowerCase()) {
+    return (
+      <NoticeWarning
+        css={styles.banner}
+        description={t('dashboard.modals.announcement.sxpDisablingBanner.description')}
+      />
+    );
+  }
+
+  // TRX migration
+  if (token.address.toLowerCase() === TOKENS.xrp.address.toLowerCase()) {
```

Comment:
This days xrp but I believe it should be trx

Reply:
Good catch!